config/clusters/leap/common.values.yaml

```diff
@@ -268,6 +276,11 @@ basehub:
           profile_options:
             image:
               display_name: Image
+              unlisted_choice:
+                enabled: True
+                display_name: "Custom org/image:tag"
```

Comment:
Let's say 'custom image' here, and use the regex validation feature to ensure it has at least a `:` in it (so we know they are specifying a tag?)